mkdocs.yml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+site_name: 'Semver Gradle Plugin'
+site_description: 'Gradle plugin to automatically calculate the next semantic version based on the git history.'
+site_author: 'Figure Technologies'
+remote_branch: gh-pages
+
+repo_name: 'gradle-semver-plugin'
+repo_url: 'https://github.com/figuretechnologies/gradle-semver-plugin'
+
+copyright: 'Copyright &copy; 2024 Figure Technologies'
+
+theme:
+  name: material
+  language: 'en'
+  favicon: 'assets/package-white.svg'
+  logo: 'assets/package-white.svg'
+  font:
+    code: 'Robot Mono'
```

Comment:
I believe the font is called `Roboto Mono` 🤷🏻‍♂️

Reply:
Omg... 😅 Great catch! I'll have to fix that!